CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## v0.x.x Unreleased
 
 ### New features
+* Added `diagonal`, `point_estimate`, `point_estimate_kwargs`, `point_estimate_marker_kwargs`, `scatter_kwargs`, `kde_kwargs` and `hexbin_kwargs` to `plot_pair` #1079
```

Comment:
I would focus more on the feature, even more so when it will probably be used constantly by most users. They can always check the api to see exactly what arguments have changed